Formula/bluepill.rb

```diff
@@ -0,0 +1,15 @@
+class Bluepill < Formula
+  desc "Bluepill is a reliable iOS testing tool that runs UI tests using multiple simulators on a single machine"
+  homepage "https://github.com/linkedin/bluepill"
+  url "https://github.com/linkedin/bluepill/releases/download/v0.2.2/bluepill-v0.2.2.zip"
+  sha256 "a487c1f36967aef3187710ae68ebd0b9d21427406c9e51fe410a73cded4a6dc9"
+  head "https://github.com/linkedin/bluepill.git"
+
+  def install
+    bin.install("bluepill")
```

Comment:
Is there no build step here? `Homebrew` prefers to build from source code instead of distributing binaries.